doc: improve prometheus docs (#2617)
woodpecker-ci/woodpecker: docs/docs/30-administration/90-prometheus.md

# Prometheus

> Woodpecker is compatible with Prometheus and exposes a `/metrics` endpoint if you set the `WOODPECKER_PROMETHEUS_AUTH_TOKEN` environment variable. Please note that access to the metrics endpoint is restricted and requires an authorization token with administrative privileges.

Reviewer: ```suggestion
Woodpecker is compatible with Prometheus and exposes a `/metrics` endpoint if the environment variable `WOODPECKER_PROMETHEUS_AUTH_TOKEN` is set. Please note that access to the metrics endpoint is restricted and requires an authorization token with administrative privileges.
```

       - targets: ['woodpecker.domain.com']
```

> You can also read the token from a file like so:

Reviewer: ```suggestion
As an alternative, the token can also be read from a file:
```
